nui/assets/js/listener.js

```diff
@@ -66,6 +67,13 @@ window.addEventListener('message', function (event) {
     });
   }
 
+  if (action == 'setStamina') {
+    Stamina.style.display = 'block';
```

Comment:
Add checks for changing the stroke to red and flashing the icon following what's already done.